engineering_tools/carma

```diff
@@ -587,12 +600,14 @@ carma__exec() {
             --volume=/opt/carma/routes:/opt/carma/routes \
             --env="XAUTHORITY=${XAUTH}" \
             --device=/dev/dri:/dev/dri \
-            --network=host \
+            --network=$NETWORK \
             --entrypoint="$entrypoint" \
+            -p 8765:8765 \
```

Comment:
Is this port specification required?

Reply:
Not exactly. I remember it being helpful when I was trying out foxglove live. However, we can add that when we actually need it. So I removed it